### ISSUE TITLE
Create prototype of new web interface

### ISSUE THREAD

**Issue by manwar** [manwar/theweeklychallenge#122]
I am looking for quick prototype that covers everything we do currently and any new ideas that helps user.

It would be pure HTML, CSS, JavaScript, Bootstrap.

The idea is later we would convert this into template that can be used with backend web framework Dancer2.

**Comment by manwar** [comment]
@himanshisrestha is interested in this issue. 
 
If anyone has any suggestion please share here.

**Comment by prakmaurya** [comment]
I would like to collaborate on this issue. Do we have to create a new and more interactive prototype for the the weekly challenge website https://theweeklychallenge.org/ . Please correct me if I am wrong.

**Comment by manwar** [comment]
@prakmaurya yes it would be a completely new and advance user friendly interface. For now just focus on prototype as how you would want the end result to be, pure HTML page with dummy data with modern look and feel.

**Comment by kemsguy7** [comment]
@prakmaurya @manwar @himanshisrestha , i'm also interested in collaborating on this issue to bring it up to speed. If you need an extra hand, i'm available

**Comment by manwar** [comment]
@kemsguy7 @prakmaurya @himanshisrestha please check out the new issues I created today. This is going to help you in designing prototype. If you have any suggestions please feel free to share with us.

**Comment by prakmaurya** [comment]
@manwar will a "resource" page with curated learning material about perl and raku and contributions be better than having it in blog? Then, we can make blog page dedicated for anyone willing to share blogs or start a discussion.

**Comment by baimamboukar** [comment]
Hello @manwar 👋
I wish to know if this prototype will be served by the `Dancer2` backend, as defined here #126 ....

**Comment by manwar** [comment]
@baimamboukar yes it would be Dancer2 web framework.

**Comment by prakmaurya** [comment]
Currently working on submission and about us page.
